<launch>
    
    <!--- Sim Time -->
    <param name="/use_sim_time" value="false" />

    <!--- Run Rviz-->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find lego_loam)/launch/map_show.rviz" />

    <!--- MapLoader -->    
    <arg name="pcd_path"  default="$(find lego_loam)/map/finalCloud.pcd"/>
    <arg name="map_topic" default="point_map"/>

    <node pkg="lego_loam" type="mapLoader"    name="mapLoader"    output="screen">
        <param name="pcd_path" value="$(arg pcd_path)"/>
        <param name="map_topic" value="$(arg map_topic)"/>
    </node>

</launch>
